<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Generate</class>
 <widget class="QWidget" name="Generate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1739</width>
    <height>1038</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generation</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Generate{
	background-color:#1E8449 ;

}
QFrame{
background:rgba(0,0,0,0.8);
border-radius:15px;
}

QPushButton{
background:#03a9f4;
color:#fff;
border-radius:15px;
font-size:22px

}
QPushButton:hover{
color: #03a9f4;
background-color:#fff;
}
QLabel{
color:#03a9f4;
background:transparent;
font-size:30px;

}
QPlainTextEdit{
color:#fff;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>1401</width>
     <height>881</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:15px;</string>
    </property>
    <property name="text">
     <string>After Generating File Contents:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>981</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#fff;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>780</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Generate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>650</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:12px;</string>
    </property>
    <property name="text">
     <string>Enter number of Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>690</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:12px;</string>
    </property>
    <property name="text">
     <string>Enter Minimum Length:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>730</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:12px;</string>
    </property>
    <property name="text">
     <string>Enter Maximum Length:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>650</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>690</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>730</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>540</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Train</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>70</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>1011</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#F4D03F ;</string>
   </property>
   <property name="text">
    <string>SELECT FILES FOR TRAINING MODELS AND GENERATING PASSWORDS!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
